std/format.d

```diff
@@ -1909,13 +1909,13 @@ void formatValue(Writer, T, Char)(auto ref Writer w, auto ref T val, const ref F
  *
  *  `formatValue` also allows to reuse existing format specifiers:
  */
-@system unittest
+@safe unittest
 {
    struct Point
    {
        int x, y;
 
-       void toString(scope void delegate(const(char)[]) sink,
+       void toString(scope void delegate(const(char)[]) @safe sink,
```

Comment:
Technically, this wouldn't actually work with a lot of real world code (and I think std.format). This is one of my big gripes with the delegate sink to string is that attributes don't really work (and I still have to formally propose the output range overload).